src/systems/ec/ec-component/ec-component-button/ec-component-button.twig

```diff
@@ -17,7 +18,6 @@
   Blocks:
     - "link_label_block": block label of the button (when "type" == "link")
     - "button_label_block": block label of the button (when "type" == "button")
-
 #}
 
 {% include '@ecl/generic-component-button' %}
```

Comment:
It would be good to explicitly pass the arguments (like https://github.com/ec-europa/europa-component-library/blob/master/src/systems/ec/ec-component/ec-component-pager/ec-component-pager.twig)